### DISCUSSION TITLE
Precipitation

### DISCUSSION THREAD

**Discussion by lisemurberg** [ACTRIS-Data-Centre/actris-jupyter-hub#11] [General]
Discussion on precipitation started at ACTRIS-Norway 3rd Annual Meeting.

**Comment by MichaelSchulzMETNO** [comment]
Which precipitation amount should be used to construct deposition from rainwater concentration and precipitation amount? What should the user know , see in an example?
Complications found in Birkenes dataset:

- Precip is in another file than concentration for all and/or part of a period.
- Precip is missing from EBAS altogether
- Periods with no precipitation are stored as zero or missing.
- Length of measurement intervals in a station varies over long time (weekly, daily etc)
- Monthy/Weekly samples cross over month borders
- Some data seem to be doubled in the database (is that from the same instrument, or double entry in database?)
- Precipitation amount might be biased low in some samplers
